packages/ctl/ctl.js

```diff
@@ -300,6 +416,76 @@ instanceConfigCommands.add(new libCommand.Command({
 		await libLink.messages.setInstanceConfigProp.send(control, request);
 	},
 }));
+
+instanceConfigCommands.add(new libCommand.Command({
+	definition: ["edit <instance> [editor]", "Edit instance configuration", (yargs) => {
+		yargs.positional("instance", { describe: "Instance to set config on", type: "string" });
+		yargs.positional("editor", {
+			describe: "Editor to use",
+			type: "string",
+			default: "",
+		});
+	}],
+	handler: async function(args, control) {
+		let instanceId = await libCommand.resolveInstance(control, args.instance);
+		let response = await libLink.messages.getInstanceConfig.send(control, { instance_id: instanceId });
+		let tmpFile = await libFileOps.getTempFile("ctl-", "-tmp", os.tmpdir());
+		let editor = await getEditor(args.editor);
+		if (editor === -1) {
+			throw new libErrors.CommandError(`No editor avalible. Checked CLI input, EDITOR and VISUAL env vars
+							  Try "ctl master config edit <editor of choice>"`);
+		}
+		let allConfigElements = "";
```

Comment:
You are defining same variable 2 lines bellow, that also makes tests fail :)
Don't forget to run linters and tests locally to verify that things work ;)

Reply:
Thanks for the catch. As I specified in my commit message, I was unfortunately unable to run tests before committing this, so i’ll fix that up asap.